spec/support/load_and_extend_graph_from_file.rb

```diff
@@ -0,0 +1,7 @@
+def load_and_extend_graph_from_file(filename)
```

Comment:
I'm usually starting with this and create modules when things get messy.

http://ryanbigg.com/2013/02/about-spec-support/